CHANGELOG.rst

```diff
@@ -0,0 +1,52 @@
+OMEGO version history
+=====================
+
+0.3.0 (February 2016)
+---------------------
+
+First large refactoring which reduces
+support for OMERO 5.1 and earlier though
```

Comment:
I'd go for "...earlier, although…" here but other than that, it all reads fine now and conforms to style guidance :+1: